workflows/train_bandit.py

```diff
@@ -154,6 +155,11 @@ def main(args):
         experiment_specific_params = get_experiment_config_from_bandit_app(
             args.experiment_id
         )
+        # in transit this gets dumped as a string so load it
+        experiment_specific_params["reward_function"] = json.loads(
+            experiment_specific_params["reward_function"]
```

Comment:
should we ideally json.loads before storing in DB in gradient-app?

Reply:
that's actually not the issue, it is loaded when pulled from the DB automatically. In transit nested dicts are passed as strings.